railties/lib/rails/testing/capybara.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+require "rails/testing/capybara_extensions"
+
+Rails.configuration.after_initialize do
+  Capybara::Selector.all.each do |name, _|
```

Comment:
Ideally, this would be a configurable list that defaults to all selectors. Since this file is required from `railties/lib/rails/test_help.rb`, I wasn't sure how to defer this code to be late enough to read from a configuration value. I would appreciate any guidance on how to achieve that.